core/server/users.js

```diff
@@ -18,7 +18,11 @@ Meteor.publish('users', function (levelId) {
     };
   }
 
-  return Meteor.users.find(filters, { fields: mainFields });
+  const users = Meteor.users.find(filters, { fields: mainFields });
+  const guildIds = users.map(u => u.guildId).filter(Boolean);
+  const guilds = Guilds.find({ _id: { $in: [...new Set(guildIds)] } });
+
+  return [users, guilds];
```

Comment:
Why do we need this?

Reply:
Without publishing the guilds, it wouldn't be possible to request the Guilds collection on the client side (`Guilds.findOne({ _id: user.guildId })?.icon` in `user-manager.js` l.74)